1907-07-01.xml

```diff
@@ -0,0 +1,1573 @@
+                                <cell><date when="1907-07-18">July 18</date></cell>
+                            </row>
+                        </table>
+                        <p>FARES FROM: Port-Said to Marseilles £7: to London and Liverpool £12; London -- Add £1 --
+                            to above fare for passengers from Cario, Ismallia, or Buez. Reduced rates on steamers
+                            not carrying vurgeon and stewardess.</p>
+                        <p> Agents in Cairo, Messrs. Thos. Cook &amp; Son. Port-Said, Messrs. Cory Brothers
+                            &amp; Co., Ltd.</p>
+                        <p>For further particulars of Freight or Passage apply to G. BEYTS &amp; Co., Suez.
+                            <measure type="indexNo">31-12-905</measure></p>
+                    </div>
+                    <div type="advert" xml:id="deg-ad-doa02">
+                        <head>Deutsche Ost Afrika Linie.</head>
+                        <p>Regelmässiger Reichspostdampferdienst. German E. African Line. Imperial Service.
+                            Departures from Port Said (Approximate Dates.)</p>
+                        <p>OUT to Aden, Zanzibar, Cape Town and intermediate ports:--</p>
+                        <p><name>S. S. Feldmarschall</name>
+                            <date when="1907-07-19">July 19</date> | <name>S. S. Kronprinz</name>
+                            <date when="1907-08-09">August 09</date></p>
+                        <p>HOME to Naples, Genoa, Marseilles, Algiers, Tangiers, Lisbon, Dover, Flushing,
+                            and Hamburg.</p>
+                        <p><name>S. S. Gonverneur</name>
+                            <date when="1907-07-06">July 06</date> | <name>S. S. Admiral</name>
+                            <date when="1907-07-13">July 13</date></p>
+                        <p>First class steamers fitted with latest improvements. Stewardesses and doctors
+                            carried. Low passage rates.</p>
+                        <p>Splendid accommodation for passengars of all classes.</p>
+                        <p>For all particulars apply to <orgName>Fix &amp; David</orgName>, CAIRO (Sharia Mansour Pacha,
+                            Telephone 865).</p>
+                    </div>
+                    <div type="advert" xml:id="deg-ad-mma01">
+                        <head>Messageries Maritimes.</head>
+                        <p>From Alexandria</p>
+                        <table rows="12" cols="5">
+                            <head><hi rend="bold">Sailing from Alexandria in June, 1907.</hi></head>
+                            <row>
+                                <cell cols="5"><hi rend="bold">For Marseilles direct</hi></cell>
+                            </row>
+                            <row>
+                                <cell>Friday</cell>
+                                <cell><date when="1905-07-05">05 July</date></cell>
```

Comment:
There are still a few dates (such as this one) in 1904 1905 and 1906. Find them using `//date/@when` xpath search in oxygen, and correct them.